### PR TITLE
Bits and pieces related to CGM^2 data reduction

### PR DESCRIPTION
**Pull request by profxj**
A few relatively minor edits:

- coadding bug-fix + new CR finder (not so successful)
- Modify lacosmic to allow for very small CR's (previously ignored)
- A few odds and ends

Run successfully on shane_kast_blue in devsuite.

### CODE REVIEW

File: pypit/arcoadd.py [hunk 1]
```diff
@@ -438,6 +445,58 @@ def scale_spectra(spectra, smask, sn2, iref=0, scale_method='auto', hand_scale=N
...
+        ax.plot(x,y)
+        # Masked
+        ax.scatter(all_w[srt][goodp][mask==1], all_f[srt][goodp][mask==1], color='r')
+        # Range
+        stdf = np.std(all_f[srt][goodp])
+        ax.set_ylim(-2*stdf, 3*stdf)
+        plt.show()
+        debugger.set_trace()
+
+    debugger.set_trace()
```
Comment (tiffanyhsyu) [inline pypit/arcoadd.py]: Should this debugger be removed?

**Comment by tiffanyhsyu** [comment]
This looks good to me!

File: pypit/arcoadd.py (hunk 1 above)
Comment (profxj) [reply pypit/arcoadd.py]: removed, although this method is not really so good..

**Comment by profxj** [comment]
@tiffanyhsyu -- You need to click on approve for me to be able to merge.
Thanks!

**Comment by profxj** [comment]
Passing the dev suite.

Merging.

**Pull request merged**